patients/repo.go

```diff
@@ -1080,6 +1053,51 @@ func cmpToMongoFilter(cmp *string) (string, bool) {
 	return f, ok
 }
 
+func PatientsToTideResult(patientsList []*Patient, period string, exclusions *[]primitive.ObjectID) []TideResultPatient {
```

Comment:
I probably missed this in the previous review, but this type juggling doesn't belong to the persistence layer.

Reply:
It was put here due to circular dependency issues, but I can refactor to move more into the api layer I think.

Reply:
As discussed offline, this code ill be remove in future iterations of the Tide dashboard